Dockerfile.keyshare.Dockerfile

```diff
@@ -1,15 +1,15 @@
 FROM openjdk:8-alpine
 
 # context: ../
-WORKDIR /app
+WORKDIR /irma_keyshare_server
```

Comment:
I am rolling back to `irma_keyshare_server` as WORKDIR because it is required by irma_keyshare_server and would be too much work to change it, then we can use the expected directory.

`/app` created an unworkable keyshare container.